OCExtensions/Classes/OCUtils.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  OCUtils.swift
+//  Pods
+//
+//  Created by Henrique Morbin on 03/08/16.
+//
+//
+
+import UIKit
+
+struct OCUtils {
```

Comment:
Since this is a Util class and can have a lot of methods, i think that we could call this like this `OCViewControllerUtils`. This way when the user calls the `OCViewControllerUtils` he knows that all methods are relative to a `ViewController`. What do you think?